tools/src/test/java/org/apache/rocketmq/tools/command/broker/GetMasterSlaveDiffCommandTest.java

```diff
@@ -0,0 +1,92 @@
+import org.apache.rocketmq.client.exception.MQClientException;
+import org.apache.rocketmq.common.protocol.body.ClusterInfo;
+import org.apache.rocketmq.common.protocol.route.BrokerData;
+import org.apache.rocketmq.remoting.exception.RemotingException;
+import org.apache.rocketmq.srvutil.ServerUtil;
+import org.apache.rocketmq.tools.admin.DefaultMQAdminExt;
+import org.apache.rocketmq.tools.command.SubCommandException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.internal.util.collections.Sets;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Set;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+
+public class GetMasterSlaveDiffCommandTest {
+
+    private GetMasterSlaveDiffCommand cmd;
+
+    private static DefaultMQAdminExt defaultMQAdminExt;
+
+    private static final String CLUSTER_NAME = "DefaultCluster";
+
+    private static final String BROKER_NAME = "broker-a";
+
+    @Before
+    public void before() throws NoSuchFieldException, IllegalAccessException, RemotingException, MQBrokerException,
+            InterruptedException, MQClientException {
+
+        ClusterInfo clusterInfo = new ClusterInfo();
+        HashMap<String, Set<String>> clusterMap = new HashMap<>(4);
+        clusterMap.put(CLUSTER_NAME, Sets.newSet(BROKER_NAME));
+        clusterInfo.setClusterAddrTable(clusterMap);
+
+        HashMap<Long, String> brokerMap = new HashMap<>(4);
+        brokerMap.put(0L, "127.0.0.1:10911");
+        BrokerData brokerData = new BrokerData(CLUSTER_NAME, BROKER_NAME, brokerMap);
+        HashMap<String, BrokerData> brokerAddrTable = new HashMap<>();
+        brokerAddrTable.put(BROKER_NAME, brokerData);
+        clusterInfo.setBrokerAddrTable(brokerAddrTable);
+
+        defaultMQAdminExt = new DefaultMQAdminExt();
+        defaultMQAdminExt = spy(defaultMQAdminExt);
+
+        cmd = new GetMasterSlaveDiffCommand();
+        Field field = cmd.getClass().getDeclaredField("defaultMQAdminExt");
+        field.setAccessible(true);
+        field.set(cmd, defaultMQAdminExt);
+
+        doReturn(clusterInfo).when(defaultMQAdminExt).examineBrokerClusterInfo();
+        doReturn(1024L).when(defaultMQAdminExt).getMasterSlaveDiff(anyString());
+    }
+
+    @After
+    public void after() {}
+
+    @Test
+    public void testExecute() throws SubCommandException, RemotingException, MQBrokerException, InterruptedException, MQClientException {
+        Options options = ServerUtil.buildCommandlineOptions(new Options());
+        String[] subargs = new String[] {"-b " + BROKER_NAME , "-c " + CLUSTER_NAME};
+        final CommandLine commandLine =
+            ServerUtil.parseCmdLine("mqadmin " + cmd.commandName(), subargs, cmd.buildCommandlineOptions(options), new PosixParser());
+        cmd.execute(commandLine, options, null);
```

Comment:
Any assert for cmd execute?

Reply:
Hi, @vongosling 
Thanks for your review, in package `org.apache.rocketmq.tools.command.broker`, none of the classes has assert for cmd execute.
For consistency of code style, I didn't add assert before

But just now, I've added the assert for cmd execute, please review again. Thank you very much.